README.md

```diff
@@ -1,86 +1,120 @@
+├── config.cfg          - the app's configuration file
+├── db_populate.py      - add initial values in to the database(mainly for testing)
+├── db_reset.py         - reset and repopulate the database
+├── error_log           - contains error log file
+├── lemur
+│   ├── __init__.py     - initialize the app and set up global variables
+│   ├── static
+│   │   ├── css         - contain css files for templates' decoration
+│   │   └── js          - contain javascripts for templates, library, utility functions or global constants
+│   ├── templates       - contains html templates
+│   ├── utility.py      - utility functions used to make view functions in views.py more readable
+│   └── views.py        - view functions for all the templates
+├── migrations          - contains files for database migration
+├── requirements.txt    - required modules/libraries for the app
+├── run.py              - the script used to migrate/upgrade the database
+└── unit_test.py        - unit test for the app
+```
+##Part3 Setup
 ###Step1 Create Virtual Environment for All Required Dependencies
 ``` mkvirtualenv lemur --python=python3```
 
-###Step2 Install Required Packages: Flask, psycopg2, SQLAlchemy, WTForms, Flask-WTF
+###Step2 Install Required Packages
 ```
-pip install -r requirement
+pip install -r requirements.txt
 ```
 
 ###Step3: Create Local Database
-We’ll need to get a Postgres database set up to store our todo list. We’ll also add Python ORM SQLAlchemy to our app. Once you have Postgres installed, create a database and name it lemur to use as a local database.<br\>
-* Run PostgreSQL command line client.<br\>
-* Create a database user with a password.<br\>
-* Create a database instance.<br\>
+We’ll need to get a Postgres database set up to store our todo list. We’ll also add Python ORM SQLAlchemy to our app. Once you have Postgres installed, create a database and name it lemur to use as a local database. <br>
+* Run PostgreSQL command line client.
+* Create a database user with a password.
+* Create two database instances(one for the app and the other for testing).
 
 ```
 psql
 create user zzy with password 'mypassword';
 create database lemur owner zzy encoding 'utf-8'; 
+create database lemur_test owner zzy encoding 'utf-8'; 
 ```
 
-###Step4: Download and Link boostrap library, jquery-ui 1.11.4 library, and jquery-2.1.3 library
+###Step4: Setup Database
+In /lemur
+```
+python3 db_create
+```
 
-###Step5: Run the app `python3 myapp`
+###Step5: Ask zhongzi@reed.edu for the private configuaration file
+In order to run the latest version
+You need to create a file named ’secret_key.cfg’ and put it into the root directory of the app
+Its content should look like this
+```
+# Secret Key file
+[key]
+SECRET_KEY = AAA
+STUDENT_API_URL = BBB
+CLASS_API_URL = CCC
+```
 
-###Step6: Set up tables in the database by entering `http://127.0.0.1:5000/admin/bootstrap` in the address line and click `Return`
+###Step6: Run the app `python3 __main__.py`
```

Comment:
oh, my b, they are at different header lvels. I still think you're better off with a list than with headers.